CONTRIBUTING.md

```diff
@@ -110,6 +110,8 @@ If you want your changes to be deployed to [production pipeline](https://dev.azu
 - Once your PR is merged and [relevant build](https://dev.azure.com/azure-sdk/internal/_build?definitionId=1580&_a=summary) completed, approve an [npm release](https://dev.azure.com/azure-sdk/internal/_release?_a=releases&view=mine&definitionId=80) from the build.
   - Note that sometimes the npm release may report failure even when it succeeded. This is because sometimes it tries to publish package twice and succeeds only on the first try. You can verify your updated npm packages were published by reviewing your
   version is on npm. See [README `packages` section](https://github.com/Azure/azure-openapi-validator#packages). You can also look at the release build log.
+- **IMPORTANT**: changes to the AutoRest extension package require additional code updates to `openapi-alps` ADO repository, and deployeent of them. Work with this tool owner to apply these steps. 
```

Comment:
```suggestion
- **IMPORTANT**: changes to the AutoRest extension package require additional code updates to `openapi-alps` ADO repository, and deployment of them. Work with this tool owner to apply these steps. 
```

Reply:
https://github.com/Azure/azure-openapi-validator/pull/519

Thx for catching this :)